motr/idx.h

```diff
@@ -46,7 +46,10 @@
  *     - DEL: given a set of keys, delete the matching records from the index;
  *
  *     - NEXT: given a set of keys, return the records with the next (in the
- *       ascending key order) keys from the index.
+ *       ascending key order) keys from the index. If the user doesn't specify 
+ *       the start key, then start key is treated as the smallest possible key
```

Comment:
I haven't looked at the rest of the documentation so my comment maybe can be ignored.  Does the word "smallest" make sense for keys?  It talks about "ascending" so perhaps "lowest" would be better.

Reply:
Thanks John. I used "smallest" because in [client.h](https://github.com/Seagate/cortx-motr/blob/e822f3af0eab46fc76cb52c6274b94cb5eaa3022/motr/client.h#L1445-L1458) they used "smallest".